jlm/tooling/CommandGraphGenerator.cpp

```diff
@@ -126,9 +127,10 @@ JlcCommandGraphGenerator::GenerateCommandGraph(const JlcCommandLineOptions & com
 
     if (compilation.RequiresOptimization())
     {
+      util::filepath tempDirectory(std::filesystem::temp_directory_path());
       auto clangCommand = util::AssertedCast<ClangCommand>(&lastNode->GetCommand());
       auto statisticsFilePath = util::StatisticsCollectorSettings::CreateUniqueStatisticsFile(
-          util::filepath(std::filesystem::temp_directory_path()),
+          util::filepath(tempDirectory),
```

Comment:
Do you need to wrap this in an additional `util::filepath`? `tempDirectory` already has that type.

Reply:
Fixed.